app/activity/[id]/page.tsx

```diff
@@ -43,7 +42,6 @@ export default function ActivityDetails() {
 
   return (
     <>
```

Comment:
You don't need React fragment as well! Please check with the other pages too

Reply:
I removed unnecessary fragments!